ui/components/src/model/units/bytes.ts

```diff
@@ -37,37 +38,52 @@ export const BYTES_UNIT_CONFIG: Readonly<Record<BytesUnitKind, UnitConfig>> = {
   },
 };
 
-export function formatBytes(bytes: number, { abbreviate, decimal_places }: BytesUnitOptions) {
-  if (bytes === 0) return '0 bytes';
+export function formatBytes(bytes: number, options: BytesUnitOptions) {
+  const { abbreviate, decimal_places } = options;
 
-  decimal_places = decimal_places ?? DEFAULT_DECIMAL_PLACES;
-  // Avoids maximumFractionDigits value is out of range error. Possible values are 0 to 20.
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumfractiondigits
-  if (decimal_places < 0) {
-    decimal_places = 0;
-  } else if (decimal_places > 20) {
-    decimal_places = 20;
+  const showFullNumber = abbreviate === false;
```

Comment:
Optional nit: Is `abbreviate` always a boolean value? If so, `!abbreviate` feels slightly cleaner here.

Reply:
Ah, I wanted to capture that it's explicitly set to `false` -- in other words, `undefined`  or `null` does not mean we should show the full number. 

It's because "abbreviating" is actually the common behavior, and not abbreviating is the uncommon behavior, so it's making sure the user explicitly asks for it. 

In an ideal world, I think it would have been cool if the parameter was `showFullNumber` (i.e. do not abbreviate) or something like that because then we could assume false/undefined to mean the same thing.